example/pubspec.yaml

```diff
@@ -18,21 +18,20 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.18.0 <4.0.0"
+  sdk: ">=3.0.0 <4.0.0"
+  flutter: ">=3.10.0"
 
 dependencies:
   flutter:
     sdk: flutter
+  vector_tile_renderer: ^4.0.0
```

Comment:
This was added because of a new lint rule which recommends directly depending on all used libraries, even if it is exposed/transitively depended on.